lib/rspecq/worker.rb

```diff
@@ -107,7 +112,8 @@ def work
           RSpec.configuration.add_formatter(Formatters::JobTimingRecorder.new(queue, job))
         end
 
-        opts = RSpec::Core::ConfigurationOptions.new(["--format", "progress", job])
+        args = [*rspec_args, "--format", "progress", job]
```

Comment:
Now that we accept a `--format` argument, having an additional, mandatory `--format progress` doesn't seem right. Could we, instead of setting it here, set it in `bin/rspecq` as a default value?